docs/ops/doc/Concepts.md

```diff
@@ -3,7 +3,7 @@
 This page is designed to help *Ops users* understand the SciJava Ops concepts powering the framework.
 
 ## Ops
-An **algorithm** is a mathematical routine that transforms, interrogate, or refines input values into output values. Algorithms are used throughout scientific computing, and the fundamental [purpose](Purpose) of SciJava Ops is to facilitate their application.
```

Comment:
I don't know Sphinx super well but when I was playing with it locally it seemed like file extensions were not necessary for links to work. I favored leaving them absent, e.g. if we change a `.rst` to an `.md` will it break? Or will it not since it's all being built as HTML anyway, in which case why include them?

Just wondering if there's a definitive reason to include them?

Reply:
Good catch - removed it!